### ISSUE TITLE
Duplicated interface names can cause problems

### ISSUE THREAD

**Issue by mpeuster** [sonata-nfv/son-emu#176]
Using the same interface names inside the VNF containers can lead to problems with VNFs that need to run in `privileged` mode, like the vTC (PF_RING). 

A workaround is to modify the VNFDs to not have duplicated interface names (not have `input` or `output` twice.

We should think about having a better naming scheme (e.g. include the vnf_id in the interface name). Not sure how long these names are allowed to be?

This is an issue that should be investigated after the Y1 demo.

**Comment by stevenvanrossem** [comment]
Interface names are defined in the VNFD, and are the developer's responsability?
Maybe the best the emulator can do is to print a warning when it detects duplicated names?

We cannot change the interface names defined in the VNFD when deploying in the emulator,
it might break the VNF functionality?

**Comment by mpeuster** [comment]
Yes, thats true. Developers need to rely on their own interface name definitions. In addition, the bug only occurs with very specific VNF implementations. Maybe a simple warning message is really the best choice here.

**Comment by mpeuster** [comment]
This is also a problem when using the same interface names for two VNFs deployed using OSM r4. So the issue is still relevant!
